public/js/mixes/index.controller.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+  (function(){
+    angular
+    .module("mixes")
+    .filter("unsafe", ["$sce", function($sce){
+      return $sce.trustasHtml
+    }])
+    .controller("MixIndexController", [
+      "$firebaseArray",
+      MixIndexControllerFunction
+    ]);
+
+    function MixIndexControllerFunction($firebaseArray){
+      // this.break = function(string){
+      //   return string.replace(/NEWLINE/g, "<br>")
+      //
+      // }
+      var vm = this;
+      var ref = firebase.database().ref().child("mixes");
+      vm.mixes = $firebaseArray(ref);
```

Comment:
if you wanted to modify each mix that came back from the database you could do it here, like:
```js
 $firebaseArray(ref).$loaded().then(function (mixes) {
      mixes.forEach(function(mix) {
             // replace each the NEWLINE reference in each mix with a `<br>` tag
      })
}) 
```